packages/cyberstorm-forms/src/forms/CreateTeamForm.tsx

```diff
@@ -12,14 +12,20 @@ import {
   useFormToaster,
 } from "@thunderstore/cyberstorm-forms";
 
-export function CreateTeamForm() {
-  const toaster = useFormToaster({
+export function CreateTeamForm(props: {
+  dialogOnChange: (v: boolean) => void;
```

Comment:
Since the form doesn't open the dialog, you could keep this interface as small as necessary with something like `closeDialog: () => void;` and the call the component with ` <CreateTeamForm closeDialog={() => setOpenDialog(false)} />`